pom.xml

```diff
@@ -59,8 +53,40 @@
     <maven.gpg.plugin.version>1.6</maven.gpg.plugin.version>
   </properties>
 
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <repository>
+      <id>ossrh</id>
```

Comment:
Similar to above, this used to be `sonatype-nexus-staging` and is referred to when we configure the `nexus-staging-maven-plugin` plugin below. Whatever value we choose needs to be the same there, here and in the settings.xml on Jenkins.